<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>311</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>301</width>
       <height>389</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="laBel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Surname:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="Surname"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Patronymic:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="Patronymic"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Country:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="country"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Team:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="team"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Game number:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="gameNumber"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Age:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="Age"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Growth:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="Growth"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Weight:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="Weight"/>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="change_in_list">
        <property name="text">
         <string>change in list</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QPushButton" name="add_in_list">
        <property name="text">
         <string> add in list</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="display">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>711</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="viewFile">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>721</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>40</y>
       <width>264</width>
       <height>174</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QComboBox" name="combo_change"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="search">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QComboBox" name="combo_delete"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="delete_2">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="sort">
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="combo_sort">
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Surname</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Patronymic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Country</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Team</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Game number</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Age</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Growth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weight</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="change">
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="combo_search">
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Surname</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Patronymic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Country</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Team</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Game number</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Age</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Growth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weight</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Go Back</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QLineEdit" name="search_"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="resave">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>380</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resave in File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="read">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>340</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>300</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>300</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1199</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
